low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -118,18 +118,18 @@ def validate_dependencies(setup):
             "zope.interface": "5.2.0"
             }
         rpm_3ps_packages = {
-            "hdparm": "9.43",
+            "hdparm": "9.54",
             "ipmitool": "1.8.18",
-            "lshw": "B.02.18",
-            "python3": "3.6.8",
+            "lshw": "B.02.19.2",
+            "python36": "3.6.8",
             "python36-dbus": "1.2.4",
             "python36-gobject": "3.22.0",
             "python36-paramiko": "2.1.1",
             "python36-psutil": "5.6.7",
             "shadow-utils": "4.6",
-            "smartmontools": "7.0",
-            "systemd-python36": "1.0.0",
-            "udisks2": "2.8.4"
+            "smartmontools": "7.1",
+            "python3-systemd": "234",
+            "udisks2": "2.9.0"
```

Comment:
These changes are also not cross-compatible, so should we have a separate pkg validation list centos7.x and rocky8.4, basically for el7 and el8 ?

Reply:
yes we should have separate list

Reply:
addressed already.